types/lumino-js-sdk.d.ts

```diff
@@ -1,200 +1,199 @@
 declare module 'lumino-js-sdk' {
+    export interface ChannelIdentifier { tokenAddress: string, partnerAddress: string }
+    export interface OpenChannelRequest { tokenAddress: string, amountOnWei: number, rskPartnerAddress?: string, rnsPartnerAddress?: string }
+    export interface BalanceIdentifier { tokenAddress: string, amountOnWei: number, partnerAddress: string }
```

Comment:
I added this to prevent some verbosity and repeated code, I think I fixed [RLP-1090](https://jirainfuy.atlassian.net/browse/RLP-1090) accidentally.

Anyway, I done like those names. 'Identifier' is not so wrong for `ChannelIdentifier`, because a channel is identified by its token and partner, but anyway sound ambiguous.

Some tip?